roles/openshift_health_checker/openshift_checks/logging/podlogs.py

```diff
@@ -0,0 +1,174 @@
+"""
+Module for ensuring logs from pods can be queried in a reasonable amount of time.
+"""
+
+import json
+import time
+
+from uuid import uuid4
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+ES_CMD_TIMEOUT_SECONDS = 600
+ES_DEFAULT_HOST = "logging-es"
+ES_DEFAULT_INDEX = "project.logging*"
+
+
+class PodLogQueryTime(LoggingCheck):
+    """Check that logs from ElasticSearch pods can be queried within a reasonable amount of time."""
+
+    name = "pod_log_query_time"
```

Comment:
Shouldn't generally be querying that takes the time, but rather waiting for the log entry to get there so you can query it. Suggest `name = "logging_index_time"`